services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/unit_templates_manager/unitTemplateMinisTable.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react'
+
+import { DataTable } from '../../../../Shared';
+import { READING_TEXTS, READING_FOR_EVIDENCE, CONNECT, DIAGNOSTIC, GRAMMAR, PROOFREADER, LESSONS } from '../assignmentFlowConstants';
+import { UnitTemplateInterface } from '../../../../../interfaces/unitTemplate';
+import { Activity } from '../../../../../interfaces/activity';
+
+export const UnitTemplateMinisTable = ({ unitTemplates }) => {
+  const toolColors = {
+    [READING_FOR_EVIDENCE]: '#2C7F9B',
+    [CONNECT]: '#DF9E3D',
+    [DIAGNOSTIC]: '#EB4F47',
+    [GRAMMAR]: '#9035D6',
+    [PROOFREADER]: '#4D8DD9',
+    [LESSONS]: '#AD287B'
+  }
+  const dataTableFields = [
+    { name: "Pack type", attribute:"packType", width: "160px", rowSectionClassName: 'pack-type-section', noTooltip: true },
+    { name: "Pack name", attribute:"packName", width: "190px", rowSectionClassName: 'pack-name-section', noTooltip: true },
+    { name: "Tools", attribute:"tools", width: "140px", rowSectionClassName: '', noTooltip: true },
+    { name: "Grade Level", attribute:"gradeLevel", width: "70px", rowSectionClassName: '', noTooltip: true },
+    { name: "Activities", attribute:"activities", width: "50px", rowSectionClassName: 'activities-section', noTooltip: true },
+    { name: "Duration", attribute:"duration", width: "150px", rowSectionClassName: '', noTooltip: true },
+  ];
+
+  function getDurationElement(unit_template_category, time) {
+    const { name } = unit_template_category
+    if(name === READING_TEXTS) {
+      return(
+        <section className="duration-section">
+          <p>{`${time} mins`}</p>
+          <section className="tags-section">
+            <p className="duration-beta-tag">BETA</p>
+            <p className="duration-new-tag">NEW</p>
```

Comment:
I'm sure that this is just the design, but I'm just a bit worried that the "NEW" part of this is going to still be there in a year.  Oh, well.  If it really becomes an issue, it'll at least be easy to remove later.  No change needed.  Compare to the dynamic date comparison in `unit_template_first_row`